libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -794,7 +1335,6 @@ Int_t TAngularCorrelation::GenerateMaps(Int_t detectors, Int_t distance)
    return val;
 }
 
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
Why was this line dropped? I think it's necessary for the doxygen documentation.